array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,11 @@
+/* exported take */
+function take(array, count) {
+  const newArray = [];
+  for (let i = 0; i < array.length; i++) {
```

Comment:
You can simplify and remove some code by adding another expression to your condition expression

```js
for (let i = 0; i < count && i < array.length; i++) {
```